app/presenters/contents_list.rb

```diff
@@ -20,7 +20,8 @@ def contents_link(text, id)
         track_category: 'contentsClicked',
         track_action: 'leftColumnH2',
         track_label: id
-      })
+      }
+)
```

Comment:
When running the linter in Jenkins it runs in [diff mode](https://github.com/alphagov/govuk-puppet/blob/7ded141088152004a54441ff36781ad7d3581737/modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy#L388), meaning it should only check lines that have changed.

This change looks wrong:
```ruby
  def contents_link(text, id)
    link_to(
      text,
      "##{id}",
      data: {
        track_category: 'contentsClicked',
        track_action: 'leftColumnH2',
        track_label: id
      }
)
  end
```